FV3/gfsphysics/GFS_layer/GFS_radiation_driver.F90

```diff
@@ -1845,6 +1845,15 @@ subroutine GFS_radiation_driver                             &
 ! mg, sfc-perts
 
 
+      if (Model%do_clearsky_rad) then
+        clouds(:,:,1) = 0.0  ! layer total cloud fraction
+        clouds(:,:,2) = 0.0  ! layer cloud liq water path
+        clouds(:,:,4) = 0.0  ! layer cloud ice water path
+        clouds(:,:,6) = 0.0  ! layer rain water path
+        clouds(:,:,8) = 0.0  ! layer snow water path
+        cldsa(:,:) = 0.0  ! fraction of clouds for low, mid, hi, tot, bl
```

Comment:
Out of curiosity, what are the 'odd' indices in this array, 3, 5, 7?

Reply:
According to comments, they are "effective radii" of the preceding tracer. But if we're setting the mixing ratio to zero, I don't think it matters what the radii are.